concept/validation/schema-validation.feature

```diff
@@ -11,490 +11,734 @@ Feature: Schema validation
     Given connection has been opened
     Given connection does not have any database
     Given connection create database: typedb
-    Given connection open schema session for database: typedb
-    Given session opens transaction of type: write
+    Given connection open schema transaction for database: typedb
 
 
   Scenario: Cyclic type hierarchies are disallowed
```

Comment:
This test can be split into 3 (even 4 with role types) tests and be put into concept files.